src/community/wps-download/src/main/java/org/geoserver/wps/gs/download/DownloadUtilities.java

```diff
@@ -50,6 +52,10 @@
     /** The LOGGER. */
     private static final Logger LOGGER = Logging.getLogger(DownloadUtilities.class);
 
+    /** Nearest Neighbour interpolation method */
```

Comment:
We don't need this, there is already an instance available in [ImageUtilities](https://github.com/geotools/geotools/blob/master/modules/library/coverage/src/main/java/org/geotools/resources/image/ImageUtilities.java#L183):